packages/jest-mock-apollo/package.json

```diff
@@ -23,21 +23,21 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/jest-mock-apollo/README.md",
   "dependencies": {
-    "apollo-link": "^1.2.3",
-    "graphql-tool-utilities": "^0.9.0",
+    "apollo-link": "^1.2.9",
+    "graphql-tool-utilities": "^0.10.0",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "apollo-cache-inmemory": "^1.2.4",
-    "apollo-client": "^2.3.4",
+    "apollo-cache-inmemory": "^1.5.1",
```

Comment:
Do we strictly need versions this high? Do lower versions have an incorrect peer dependency? I'd like us to be lenient on react/apollo/typescript dependencies where we can be

Reply:
i didn't check specifically, but i think all needed some sort of a bump. I can definitely peer into their deps and see what is the min requirement.

Reply:
managed to lower the following packages a bit:

* `apollo-cache-inmemory@^1.4.0`
* `apollo-client@^2.4.9`
* `graphql-tag@^2.10.0`
* `react-apollo@^2.2.3`

These are the bare minimum support levels for GraphQL 14